_nodes/0011.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: The full internal subcategory associated to a displayed object
+title: The full internal subfibration associated to a displayed object
```

Comment:
Not sure what happened here haha

Reply:
oh i guess this is some old commit of mine that i later undid.

Reply:
Ok, I'll just dump it then.